### PR TITLE
[TC-6712] - update attach documents docs

### PR DESCRIPTION
**Pull request by olegtradecloud**
## Description

https://tradecloud.atlassian.net/browse/TC-6712

### Scope
This PR updates attach documents docs.
Can be seen on this [Gitbook page](https://app.gitbook.com/@tradecloud/s/api/v/TC-6712-attach-documents/buyer/issue/attach-document)

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: buyer/issue/attach-document.md [hunk 1]
```diff
@@ -72,96 +72,25 @@ Document upload failed, the document size exceeds 100 MB
 [Upload document OpenAPI specs](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/object-storage/specs.yaml#/object-storage/uploadDocument)
 {% endhint %}
 
-### Step 2. Attach the uploaded document `objectId` to the order or line
+### Step 2. Choose an appropriate url to attach the document
```
Comment (marcmatt) [inline buyer/issue/attach-document.md:75]: appropriate API (3x)

**Review by marcmatt — CHANGES_REQUESTED** [review]
Looks good, some improvements needed.
- Also add to "Issue a new order" -> "Lines" -> "Other line fields":
"documents: contain attached documents, see:"
And then the link to /buyer/issue/attach-document
- This has already been done for "Other order fields"
  change "contain meta data and link of attached documents" to "documents: contain attached documents, see:"

File: buyer/issue/attach-document.md (hunk 1 above)
Comment (olegtradecloud) [reply buyer/issue/attach-document.md:75]: Fixed

File: buyer/issue/attach-document-url.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+* `code` : optional document code or number. The document code should be unique within your company and immutable.
+* `revision`: optional document revision \(or version\) code or number
+* `name` : optional document short \(file\) name
+* `description` : optional document description or long name
+* `type`: optional document business type
+* `objectId`: optional Tradecloud object identifier which was returned by the Tradecloud object-storage upload
+* `url`: optional document location if it is not stored in the Tradecloud object-storage.
+
+Body example:
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply buyer/issue/attach-document-url.md:102]: Fixed

File: buyer/issue/attach-document-url.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
+---
+description: Choose an appropriate url to attach a document to an order or line
+---
+
+# URL for a documents attaching
+
+You can attach documents using either of the following urls
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply buyer/issue/attach-document-url.md:7]: Fixed

File: buyer/issue/attach-document-url.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+## 1. Attach documents using the /order/documents API
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply buyer/issue/attach-document-url.md:9]: Fixed

File: buyer/issue/attach-document-url.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
+---
+description: Choose an appropriate url to attach a document to an order or line
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply buyer/issue/attach-document-url.md:2]: Fixed

File: SUMMARY.md [hunk 6]
```diff
@@ -36,6 +36,7 @@
   * [Issue a new order](buyer/issue/README.md)
     * [Attach a document to an order](buyer/issue/attach-document.md)
     * [No delivery expected](buyer/issue/no-delivery-expected.md)
+    * [Choose attach document URL](buyer/issue/attach-document-url.md)
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply SUMMARY.md:39]: Fixed

File: buyer/issue/attach-document-url.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply buyer/issue/attach-document-url.md]: @marcmatt It was supposed to be an example of `/order`  body, but without additional fields

**Review by marcmatt — APPROVED** [review]
Looks great!

**Pull request merged**